spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
I was hoping for a quick win without understanding this issue completely.

This is tricky problem. If I just take the test case by itself, it passes as written. If I change Ruby's default external encoding too, like you did in the rake config, I get an error in the spec itself:
```
     Encoding::UndefinedConversionError:
       U+2764 from UTF-8 to US-ASCII
     # ./spec/support/file_helper.rb:20:in `write'
     # ./spec/support/file_helper.rb:20:in `puts'
     # ./spec/support/file_helper.rb:20:in `block in create_file'
     # ./spec/support/file_helper.rb:13:in `open'
     # ./spec/support/file_helper.rb:13:in `create_file'
     # ./spec/rubocop/config_loader_spec.rb:391:in `block (3 levels) in <main>'
```

You don't get that?